action/protocol/staking/candidate_center.go

```diff
@@ -411,12 +426,13 @@ func (cc *candChange) delete(owner address.Address) {
 // candBase funcs
 //======================================
 
-func newCandBase() *candBase {
+func newCandBase(fixAlias bool) *candBase {
```

Comment:
This is a factory pattern apparently. No need to use `bool` to new a object.

Reply:
can you be more clear? what is your suggested change?

Reply:
if fixAlias {
 c.base = newCandBaseLegacy()
} else {
 c.base = newCandBase()
}